src/App.tsx

```diff
@@ -15,71 +15,105 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type State = {
+  selectedGood: string,
+};
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export class App extends React.Component<{}, State> {
+  state = {
+    selectedGood: 'Jam',
```

Comment:
Replace with `goods[goods.indexOf('Jam')] || goods[0]`.

About your code -> If you will not have the element 'Jam' - you will have some bugs.

Reply:
ok, but I think it's better to leave an empty line.
Like this goods[goods.indexOf('Jam')] || ''